examples/features/loops.effekt.md

```diff
@@ -0,0 +1,46 @@
+# Loops
+
+In Effekt, there is only a `while` loop expression built-in (that is completely standard).
+
+```
+def fib(n: Int): Int = {
+  var last = 0
+  var current = 1
+  var iters = 0
+  while (iters < n) {
+    val next = last + current
+    last = current
+    current = next
+    iters = iters + 1
+  } else { println("ran for " ++ show(iters) ++ " iterations") }
+  last
+}
+```
+
+```repl
+fib(10)
+```
+
+A `while` loop consists a condition on which to continue the loop and an optional `else` branch that is run when the
+`while` loop ended.
+
+Furthermore, there is a `loop` function defined in the standard library, you can use for infinite, breakable loops.
```

Comment:
I think it's also worth it to show `each` / `repeat` here :)
I was also thinking about `.foreach` on lists, as that's a looping operation I use _all the time_, but I'm not sure it fits into this chapter.